main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Get workspace-type scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting workspace-type', () => {
+    it('should fail if user is inactive', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      const workspaceTypeId = setup.gen.string({ prefix: 'workspace-test' });
+
+      await adminSession.resources.workspaceTypes.create({
+        id: workspaceTypeId,
+        status: 'approved',
+      });
+
+      await adminSession.resources.users.deactivateUser(researcherSession.user);
+
+      await expect(
+        researcherSession.resources.workspaceTypes.workspaceType(workspaceTypeId).get(),
+      ).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should fail to get no-approved types if user is not an admin', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      const workspaceTypeId = setup.gen.string({ prefix: 'workspace-test' });
+
+      await adminSession.resources.workspaceTypes.create({
+        id: workspaceTypeId,
+        status: 'not-approved',
+      });
+
+      await expect(
+        researcherSession.resources.workspaceTypes.workspaceType(workspaceTypeId).get(),
+      ).rejects.toMatchObject({
+        code: errorCode.http.code.forbidden,
+      });
+    });
+
+    it('should pass to get approved types if user is not an admin', async () => {
```

Comment:
In most of our tests description, I don't think we say "should pass", maybe we can do something like "should return approved types even if user is not an admin"